VibratoPlugin/Source/PluginProcessor.cpp

```diff
@@ -157,13 +163,13 @@ void VibratoPluginAudioProcessor::processBlock (AudioBuffer<float>& buffer, Midi
     // the samples and the outer loop is handling the channels.
     // Alternatively, you can process the samples with the channels
     // interleaved by keeping the same state.
-//    for (int channel = 0; channel < totalNumInputChannels; ++channel)
-//    {
-//        auto* channelData = buffer.getWritePointer (channel);
-//
-//        // ..do something to the data...
-//    }
 
+    float modWidthInS = *m_fParamWidthInMs / 1000;
+    float modFreqInHz = *m_fParamFreqInHz;
+    int bypassed = round(*m_fParamBypassed);
+
+    vibrato->setParam(CVibrato::kParamModWidthInS, modWidthInS * static_cast<float>(1-bypassed));
+    vibrato->setParam(CVibrato::kParamModFreqInHz, modFreqInHz);
```

Comment:
Setting params again and again for each block seems redundant. I don't think this should be done in `processBlock` (on the audio thread). In the previous implementation, the GUI thread was responsible for setting the params which would be done only when a change was made by the user.